amundsen_application/static/js/components/ProfilePage/index.tsx

```diff
@@ -37,41 +49,52 @@ export class ProfilePage extends React.Component<ProfilePageProps> {
 
   componentDidMount() {
     this.props.getUserById(this.userId);
+    this.props.getUserOwn(this.userId);
+    this.props.getUserRead(this.userId);
+    this.props.getBookmarksForUser(this.userId);
   }
 
   getUserId = () => {
     return this.userId;
   };
 
-  // TODO: consider moving logic for empty content into Tab component
-  createEmptyTabMessage = (message: string) => {
+
+  getTabContent = (resource: Resource[], label: string, source: string) => {
```

Comment:
I think the `//TODO` should be left in. While we don't have to do it now, we will save ourselves the trouble of having to have this code chunk on every page that uses `Tabs`. In addition it will make it such that the empty message logic/markup will only have to be unit tested once in `Tabs` component.